util/aggregation.py

```diff
@@ -148,34 +148,28 @@ def aggregate_provenance(job_dir, output_dir, request_id=None):
             json.dump(aggregate_dict, fh, indent=2)
 
 
-def aggregate_annotation_group(job_dir, files):
-    aggregate_dict = {'annotations': []}
-    recorded_annotation_ids = []
-    for f in sorted(files):
-        path = os.path.join(job_dir, f)
-        data = json.load(open(path))
-        for key in data:
-            if key == 'annotations':
-                new_annotations = [x for x in data[key] if x['id'] not in recorded_annotation_ids]
-                recorded_annotation_ids.extend([v['id'] for v in new_annotations])
-                aggregate_dict[key].extend(new_annotations)
-            else:
-                aggregate_dict.setdefault(key, {})[f] = data[key]
-    return aggregate_dict
-
-
 @log_timing(log)
 def aggregate_annotations(job_dir, output_dir, request_id=None):
-    groups = {}
-    anno_label = '_annotations_'
+    anno_label = 'annotations_'
+    aggregate_dict = {'annotations': []}
```

Comment:
The literal string 'annotations' is used in multiple locations. Should it be made into a named string?

Reply:
Changing to use anno_string with value 'annotations' in place of literal 'annotations'.